easybuild/easyconfigs/n/netCDF/netCDF-4.3.0-ictce-5.4.0.eb

```diff
@@ -0,0 +1,39 @@
+name = 'netCDF'
+version = '4.3.0'
+
+homepage = 'http://www.unidata.ucar.edu/software/netcdf/'
+description = """NetCDF (network Common Data Form) is a set of software libraries 
+ and machine-independent data formats that support the creation, access, and sharing of array-oriented 
+ scientific data."""
+
+toolchain = {'name': 'ictce', 'version': '5.4.0'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [
+    'ftp://ftp.unidata.ucar.edu/pub/netcdf/',
+    'ftp://ftp.unidata.ucar.edu/pub/netcdf/old',
+]
+
+patches = [('netCDF-4.3.0_nc-config.in.cmake', 'netcdf-4.3.0')]
```

Comment:
Hmm, ok, `'%(builddir)s/netcdf-%(version)s'` doesn't work either since then you get the equivalent of `%(builddir)s/%(builddir)s/netcdf-%(version)s`, damn...